sample-data/fake.js

```diff
@@ -2,119 +2,148 @@ let faker = require('faker');
 let path = require('path');
 let fs = require('fs');
 
-// for fs
-let sampleData = [];
-let writeStream = fs.createWriteStream(path.join(__dirname, './data/data.csv'));
+let cliArgs = process.argv.slice(2);
+let outputDataPath = cliArgs[0];
```

Comment:
consider `const [array, destructuring] = assignment`

Reply:
Also, you currently aren't using these closure-scope variables. You may want to set them as default values with es6 `(default = value) => syntax`